modules/mod_ginger_edit/templates/_admin_tinymce_overrides_js.tpl

```diff
@@ -1,7 +1,19 @@
 /*
 Custom settings to override tiny-init.js.
 */
-tinyInit.language="{{ m.translation.language|default:"en" }}";
-tinyInit.menubar="";
-tinyInit.toolbar="styleselect | bold italic | bullist numlist | removeformat | zlink zmedia | link unlink | code";
-tinyInit.extended_valid_elements="iframe[src|style|width|height|scrolling|marginwidth|marginheight|frameborder]";
+
+if (typeof tinymce_overrides_language === 'undefined' || !tinymce_overrides_language) {
+    tinyInit.language="{{ m.translation.language|default:"en" }}";
+}
```

Comment:
Why not just check if `tinyInit.language` itself is already set (instead of introducing a separate variable, which requires clients to set that variable, too)?